src/components/StaticShareImage.tsx

```diff
@@ -19,6 +20,7 @@ export type Props = {
   };
   topDMS: ListElement[];
   topGuilds: ListElement[];
+  t: ReturnType<typeof useTranslation>["t"];
```

Comment:
Did you test it? All good if it works

Reply:
Yes, this has been tested and works. The issue here was that I cannot import a hook inside a hook that is also inside another hook 😅. This was the solution I came with.

Reply:
Yes it's normal if you can't import it in this component I think

Reply:
Yes, this helps avoiding hook loops 😅